Dictionary.json

```diff
@@ -283,6 +283,41 @@
 		"Tips4" : "https://windowsreport.com/how-to-use-gimp-2-10/"
 	},
 
+	{	"Title" : "Go",
+		"Tag": "Programming language",
+		"Documentation" : "https://golang.org/doc/",
+		"YT1" : "https://www.youtube.com/channel/UC8butISFwT-Wl7EV0hUK0BQ",
+		"YT2" : "https://www.youtube.com/user/javaboynavin",
+		"YT3" : "https://www.youtube.com/user/Udacity",
+		"YT4" : "https://www.youtube.com/playlist?list=PLzMcBGfZo4-mtY_SE3HuzQJzuj4VlUG0q",
+		"YT5" : "https://www.youtube.com/playlist?list=PLgPJX9sVy92yu7If3I7GonlWA8YU1BuAk",
+		"Website1": "https://www.educative.io/blog/golang-tutorial",
+		"Website2": "https://golang.org/doc/tutorial/getting-started",
+		"Website3": "https://www.tutorialspoint.com/go/index.htm",
+		"Website4": "https://www.guru99.com/google-go-tutorial.html",
+		"Website5": "https://www.geeksforgeeks.org/go-programming-language-introduction/",
+		"Website6": "https://builtin.com/software-engineering-perspectives/golang-advantages",
+		"Website7": "https://yourbasic.org/golang/advantages-over-java-python/",
+		"Website8": "https://www.freecodecamp.org/news/here-are-some-amazing-advantages-of-go-that-you-dont-hear-much-about-1af99de3b23a/",
+		"Dedicated GH page1" : "https://github.com/DaoCloud/golang-mongo-sample",
+		"Dedicated GH page2": "https://github.com/kyawmyintthein/golangRestfulAPISample",
+		"Dedicated GH page3": "https://github.com/digitalocean/sample-golang",
+		"Dedicated GH page4": "https://github.com/golang/go",
+		"Dedicated GH page5": "https://github.com/golang/oauth2",
+		"Dedicated GH page6": "https://github.com/golang/gofrontend",
+		"Dedicated GH page7": "https://github.com/golang/appengine",
```

Comment:
https://github.com/avelino/awesome-go
add awesme pages also
search for awesome golang on GH and add them,